exercises/Optimization_Based_Planning/solution_ws/src/.rosinstall

```diff
@@ -0,0 +1,4 @@
+- git: {local-name: tesseract, uri: 'https://github.com/ros-industrial-consortium/trajopt_ros.git', version: kinetic-devel}
+- git: {local-name: trajopt, uri: 'https://github.com/ros-industrial-consortium/trajopt_ros.git', version: kinetic-devel}
```

Comment:
This points to branch tips, which can be moving targets. Should this point to specific tags or commits instead?

Not sure how much we value stability, but would be a shame if you spend 30 mins trying to help students fix some error which was introduced after you've verified that "everything worked" when you wrote these exercises.

Reply:
That's a good point. I just tied it to the latest commits in Tesseract, Trajopt, and iiwa_stack that are known to work for Kinetic and Melodic